README.md

```diff
@@ -61,13 +61,13 @@ A Police Computer Plugin and Server for LSPDFR.
 ## Custom Styles
 
 - File: `EPC/custom.css`
-- For simple customization you may change colors, etc. size in the `:root` selector
-- Feel free to also add more css. `EPC/custom.css` will overwrite `EPC/main/styles.css`
+- For simple customization you may change colors, etc., and size in the `:root` selector
```

Comment:
I fucked up but you did too 😅
(Either remove etc. and list everything or only list one)